heppy/Response.py

```
@@ -42,20 +42,20 @@ def put_extension_block(self, response, command, root_tag, tags_data):
+        for tag_name, attrs in tags_data.items():
             response.put_tag_data(data, root_tag, module_name + ':' + tag_name, attrs)
         response.put_to_list('extensions', data)
 
     def put_to_dict(self, name, values):
         if name not in self.data:
             self.data[name] = {}
-        for k, v in values.iteritems():
+        for k, v in values.items():
             self.data[name][k] = v
 
     def put_to_list(self, name, value=[]):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)